### ISSUE TITLE
Add option for ShortLinks to be allowed in robots.txt

### ISSUE THREAD

**Issue by MartinHotmann** [shlinkio/shlink#955]
I just noticed that in the [robots.txt](https://github.com/shlinkio/shlink/blob/develop/public/robots.txt) EVERYTHING is disallowed.

Now my situation is the following one:

I use a shortLink generated with Shlink on my Website as Menu-Link to track it and to have a short link in it. Google noticed this and crawled it. Now as the link is disallowed it warned me about that it is getting displayed on Google even it is basically blocked by the robots.txt.

Actually Google is doing right by crawling and displaying it as I want this. To not confuse Google I would like to have an option to select if the Shortlink should be crawlable or not.
If it should be crawlable (then it must not be a secret, but public link!) it should be excluded from the disallow of the robots.txt

Notice: this will reveal/expose all links that should be crawlable as they have to be mentioned in the robots.txt so if the link is not getting used publicly or is created as a secret link it should NOT be marked as crawlable.
Also: standardvalue of "crawlable" should ne `false` or `0`

The robots.txt then should be created, or overwritten so if new crawlable links are getting added later it will be updated.

Could look like this:
![Shlink Crawlable](https://user-images.githubusercontent.com/17692860/103867973-7fa2f780-50c8-11eb-8d5e-6b2e1b1df177.PNG)

![Shlink Crawlable](https://user-images.githubusercontent.com/17692860/103868019-96e1e500-50c8-11eb-88ab-9bc278b3e7bb.PNG)

**Comment by acelaya** [comment]
This is something nice to have.

The robots.txt file was added at some point as it is since it could be considered a sensitive default.

Of course it gives very little flexibility this way. Your suggestion would be useful.

**Comment by acelaya** [comment]
Feature merged. Will be part of Shlink v2.7.0, and supported on shlink-web-client 3.2.0
